daemon/gluster-blockd.c

```diff
@@ -458,6 +458,14 @@ gbDependenciesVersionCheck(void)
   }
   GB_FREE(out);
 
+  out = gbRunnerGetOutput("python -c 'from configshell_fb import __version__; print(__version__)'");
```

Comment:
Can we use the macro way ? see version.h

Reply:
Sure, will fix it.